src/main/java/org/zalando/nakadi/controller/advice/NakadiProblemExceptionHandler.java

```diff
@@ -84,6 +86,11 @@ private String generateErrorTraceId() {
         return create(Problem.valueOf(BAD_REQUEST, message), request);
     }
 
+    @ExceptionHandler(EofException.class)
+    public void handleEofException() {
+        LOG.info("Client closed connection");
```

Comment:
I don't really see much value in this log as it doesn't provide any specific information like subscription id or session id. Is it possible to have this info?

Reply:
Ok, you pointed out that FlowId will be there. Good for me.